### PR TITLE
Cypress testing

### PR DESCRIPTION
**Pull request by BryanGK**
This PR sets ups Cypress testing and includes the first e2e test.

ref #21

### CODE REVIEW

File: app/cypress/integration/end2end_spec.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+    cy.get('input[name="Postal code:"]').type('V8E7A1');
+
+    cy.contains('Continue').click();
+
+    cy.get('input[name="Primary contact (for this application):"]').type('Jane Doe');
+    cy.get('input[name="Position/title:"]').type('Applications specialist');
+    cy.get('input[name="Email:"]').type('fake@email.com');
+    cy.get('input[name="Telephone:"]').type('(555)555-1234');
+
+    cy.contains('Cancel').click();
```
Comment (wenzowski) [inline app/cypress/integration/end2end_spec.js:22]: before we cancel, don't we want to try continuing to see what happens with our inputs

File: app/cypress/integration/end2end_spec.js [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+
+    cy.location('pathname').should('eq', '/');
```
Comment (wenzowski) [inline app/cypress/integration/end2end_spec.js:24]: in addition to asserting the location, is there anything on the page we can assert for that a user would see?

**Review by wenzowski — CHANGES_REQUESTED** [review]
I'll have a bit of trouble reviewing right now since I can't spin this up...but in general let's assert for expected behaviour after we type in the fields. On a great track!

File: app/cypress/integration/end2end_spec.js (hunk 1 above)
Comment (BryanGK) [reply app/cypress/integration/end2end_spec.js:22]: At this moment nothing happens on the UI and in the console you get an unhandledRejection error and a log of the object. What's there to test?

File: app/cypress/integration/end2end_spec.js (hunk 2 above)
Comment (BryanGK) [reply app/cypress/integration/end2end_spec.js:24]: I could check for the apply button but what's the best practice in this case? Would you check for the some visual element or the path? Or both?

**Review by wenzowski — APPROVED** [review]
From standup, let's merge & pull comments into next issue about running cypress on ci

**Pull request merged**